content/zh-cn/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md

```diff
@@ -0,0 +1,328 @@
+Then, RBAC rules are created to control which PodSecurityPolicy applies to a given pod.
+If a pod meets the requirements of its PSP, it will be admitted to the cluster as usual.
+In some cases, PSP can also modify Pod fields, effectively creating new defaults for those fields.
+If a Pod does not meet the PSP requirements, it is rejected, and cannot run.
+-->
+首先，在集群中创建一个或多个 PodSecurityPolicy 资源来定义 Pod 必须满足的要求。
+然后，创建 RBAC 规则来决定为特定的 Pod 应用哪个 PodSecurityPolicy。
+如果 Pod 满足其 PSP 的要求，则照常被允许加入集群。
+在某些情况下，PSP 还可以修改 Pod 字段，有效地为这些字段创建新的默认值。
+如果 Pod 不符合 PSP 要求，则被拒绝加入集群，并且无法运行。
+
+<!--
+One more important thing to know about PodSecurityPolicy: it’s not the same as
+[PodSecurityContext](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context).
+-->
+关于 PodSecurityPolicy，还需要了解：它与
+[PodSecurityContext](zh-cn/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context) 不同。
+
+<!--
+A part of the Pod specification, PodSecurityContext (and its per-container counterpart `SecurityContext`)
+is the collection of fields that specify many of the security-relevant settings for a Pod.
+The security context dictates to the kubelet and container runtime how the Pod should actually be run.
+In contrast, the PodSecurityPolicy only constrains (or defaults) the values that may be set on the security context.
+-->
+作为 Pod 规范的一部分，PodSecurityContext（及其每个容器对应的 `SecurityContext`）
+是字段的集合，这些字段为 Pod 指定了与安全相关的许多设置。
+安全上下文指示 kubelet 和容器运行时究竟应该如何运行 Pod。
+相反，PodSecurityPolicy 仅约束可能在安全上下文中设置的值（或设置默认值）。
+
+<!--
+The deprecation of PSP does not affect PodSecurityContext in any way.
+-->
+弃用 PSP 不会以任何方式影响 PodSecurityContext。
+
+<!--
+## Why did we need PodSecurityPolicy?
+-->
+## 为什么需要 PodSecurityPolicy？
+
+<!--
+In Kubernetes, we define resources such as Deployments, StatefulSets, and Services that
+represent the building blocks of software applications. The various controllers inside
+a Kubernetes cluster react to these resources, creating further Kubernetes resources or
+configuring some software or hardware to accomplish our goals.
+-->
+在 Kubernetes 中，我们定义了 Deployment、StatefulSet 和 Service 等资源。
+这些资源代表软件应用程序的构建块。Kubernetes 集群中的各种控制器根据这些资源做出反应，
+创建更多的 Kubernetes 资源或配置一些软件或硬件来实现我们的目标。
+
+<!--
+In most Kubernetes clusters, 
+RBAC (Role-Based Access Control) [rules](/docs/reference/access-authn-authz/rbac/#role-and-clusterrole)
+control access to these resources. `list`, `get`, `create`, `edit`, and `delete` are
+the sorts of API operations that RBAC cares about, 
+but _RBAC does not consider what settings are being put into the resources it controls_.
+For example, a Pod can be almost anything from a simple webserver to
+a privileged command prompt offering full access to the underlying server node and all the data.
+It’s all the same to RBAC: a Pod is a Pod is a Pod.
+-->
+在大多数 Kubernetes 集群中，由 RBAC（基于角色的访问控制）[规则]
+(zh-cn/docs/reference/access-authn-authz/rbac/#role-and-clusterrole)
```

Comment:
Fix the link.

Reply:
> Fix the link.

I checked the en blog and found the link work well. So, I jsut added a slash at the beginning as you suggested in other comments.